packages/delivery-xml-http-request/delivery.js

```diff
@@ -1,5 +1,14 @@
 const payload = require('@bugsnag/core/lib/json-payload')
 
+function deliveryFailed (client, body, cb) {
+  const err = new Error('Event failed to send')
```

Comment:
Can we add some useful information to the message here, e.g. the status code?

Maybe something like "Request failed with status ${status}"